recipes/tip/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "tip" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/IM-USAF/tip/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4ba6f6a9e652fa48a99534997aaaa0921a94af0ef2d65ce909b6825c06a2b259
+
+build:
+  skip: true  # [win and vc<14 or osx]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make 4.*  # [unix]
+    - ninja 1.*
+    - cmake 3.*
+    - pkgconfig 1.*
+
+  host:
+    - libpcap 1.*
+    - arrow-cpp 6.*
+    - spdlog 1.*
+    - yaml-cpp 0.*
+    - gmock 1.*
+    - gtest 1.*
+    - libtins 4.*
+    - fmt 9.*
+
+  run:
+    - libpcap 1.*
+    - arrow-cpp 6.*
+    - spdlog 1.*
+    - yaml-cpp 0.*
+    - gmock 1.*
+    - gtest 1.*
+    - libtins 4.*
+    - fmt 9.*
```

Comment:
I accepted this commit and found that entries in requirements.host must be pinned to major versions to build. The suggested lines in requirements.run remain.

Reply:
> I accepted this commit and found that entries in requirements.host must be pinned to major versions to build.

What do you mean it didn't build? It looks like [the build succeeded](https://github.com/conda-forge/staged-recipes/pull/20869/commits/48bcfe9ecc6390c8077bb8eba226943e6b6d9f01).

Reply:
It didn't build locally and I didn't bother checking the CI tests. Just reconfirmed that it does not build locally without the pins, though it does build in the CI. The command I'm running locally in an env which has only conda-build and conda-verify installed, after navigating to staged-recipes/recipes, is `conda build -c conda-forge --error-overlinking tip`. Is that correct? 

Locally, it fails at the tests stage in linux and windows with (windows build output)
```
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed
failed to get install actions, retrying.  exception was: Unsatisfiable dependencies for platform win-64: {"vs2015_runtime[version='>=14.16.27012,<15.0a0|>=14.16.27033|>=14.29.30139']", "vc[version='>=14.1,<15.0a0|>=14.2,<15']", "vs2015_runtime[version='>=14.16.27033|>=14.29.30139']", "vs2015_runtime[version='>=14.27.29016|>=14.28.29325|>=14.32.31332|>=14.29.30139']", "vc[version='>=14.1,<15.0a0|>=14.1,<15|>=14.2,<15']", "vs2015_runtime[version='>=14.15.26706|>=14.16.27012|>=14.16.27033|>=14.28.29325|>=14.32.31332|>=14.29.30139|>=14.29.30037|>=14.27.29016']", "gtest[version='>=1.11.0,<2.0a0']", "arrow-cpp[version='>=6.0.2,<6.0.3.0a0|>=6.0.2,<7.0a0']", "vs2015_runtime[version='>=14.16.27033']", "vc[version='>=14.1,<15.0a0']", "vc[version='>=14.1,<15|>=14.2,<15']", "libtins[version='>=4.4,<5.0a0']", "gtest==1.11.0[build='h59b6b97_0|h2d74725_0']", "ucrt[version='>=10.0.22621.0,<11.0a0']", "vs2015_runtime[version='>=14.15.26706|>=14.16.27012|>=14.27.29016|>=14.28.29325|>=14.32.31332|>=14.29.30037']", "vs2015_runtime[version='>=14.15.26706|>=14.16.27012|>=14.16.27012,<15.0a0|>=14.29.30037|>=14.29.30139|>=14.32.31332|>=14.28.29325|>=14.27.29016']", "spdlog[version='>=1.11.0,<1.12.0a0|>=1.11.0,<2.0a0']", "ucrt[version='>=10.0.20348.0']", "vs2015_runtime[version='>=14.15.26706|>=14.16.27012|>=14.28.29325|>=14.32.31332|>=14.29.30139|>=14.29.30037|>=14.27.29016']", "gmock[version='>=1.11.0,<2.0a0']", "yaml-cpp[version='>=0.7.0,<0.8.0a0|>=0.7.0,<1.0a0']", "vs2015_runtime[version='>=14.15.26706|>=14.16.27012|>=14.32.31332|>=14.28.29325|>=14.29.30139|>=14.29.30037|>=14.27.29016']", 'ucrt', "vs2015_runtime[version='>=14.29.30139']", "fmt[version='>=9.1.0,<10.0a0']", "vc[version='>=14.1,<15|>=14.1,<15.0a0']", "vs2015_runtime[version='>=14.15.26706|>=14.16.27012|>=14.16.27012,<15.0a0|>=14.29.30037|>=14.32.31332|>=14.28.29325|>=14.27.29016']", "vc[version='>=14.1,<15']", "vs2015_runtime[version='>=14.29.30037']", "vs2015_runtime[version='>=14.16.27012,<15.0a0|>=14.16.27033']", "vs2015_runtime[version='>=14.27.29016|>=14.28.29325|>=14.29.30037|>=14.32.31332']", "vc[version='14.*|9.*|>=14.1,<15.0a0|>=14,<15.0a0|>=9,<10.0a0']", "vc[version='>=14.2,<15']"}
WARNING:conda_build.build:failed to get install actions, retrying.  exception was: Unsatisfiable dependencies for platform win-64: {"vs2015_runtime[version='>=14.16.27012,<15.0a0|>=14.16.27033|>=14.29.30139']", "vc[version='>=14.1,<15.0a0|>=14.2,<15']", "vs2015_runtime[version='>=14.16.27033|>=14.29.30139']", "vs2015_runtime[version='>=14.27.29016|>=14.28.29325|>=14.32.31332|>=14.29.30139']", "vc[version='>=14.1,<15.0a0|>=14.1,<15|>=14.2,<15']", "vs2015_runtime[version='>=14.15.26706|>=14.16.27012|>=14.16.27033|>=14.28.29325|>=14.32.31332|>=14.29.30139|>=14.29.30037|>=14.27.29016']", "gtest[version='>=1.11.0,<2.0a0']", "arrow-cpp[version='>=6.0.2,<6.0.3.0a0|>=6.0.2,<7.0a0']", "vs2015_runtime[version='>=14.16.27033']", "vc[version='>=14.1,<15.0a0']", "vc[version='>=14.1,<15|>=14.2,<15']", "libtins[version='>=4.4,<5.0a0']", "gtest==1.11.0[build='h59b6b97_0|h2d74725_0']", "ucrt[version='>=10.0.22621.0,<11.0a0']", "vs2015_runtime[version='>=14.15.26706|>=14.16.27012|>=14.27.29016|>=14.28.29325|>=14.32.31332|>=14.29.30037']", "vs2015_runtime[version='>=14.15.26706|>=14.16.27012|>=14.16.27012,<15.0a0|>=14.29.30037|>=14.29.30139|>=14.32.31332|>=14.28.29325|>=14.27.29016']", "spdlog[version='>=1.11.0,<1.12.0a0|>=1.11.0,<2.0a0']", "ucrt[version='>=10.0.20348.0']", "vs2015_runtime[version='>=14.15.26706|>=14.16.27012|>=14.28.29325|>=14.32.31332|>=14.29.30139|>=14.29.30037|>=14.27.29016']", "gmock[version='>=1.11.0,<2.0a0']", "yaml-cpp[version='>=0.7.0,<0.8.0a0|>=0.7.0,<1.0a0']", "vs2015_runtime[version='>=14.15.26706|>=14.16.27012|>=14.32.31332|>=14.28.29325|>=14.29.30139|>=14.29.30037|>=14.27.29016']", 'ucrt', "vs2015_runtime[version='>=14.29.30139']", "fmt[version='>=9.1.0,<10.0a0']", "vc[version='>=14.1,<15|>=14.1,<15.0a0']", "vs2015_runtime[version='>=14.15.26706|>=14.16.27012|>=14.16.27012,<15.0a0|>=14.29.30037|>=14.32.31332|>=14.28.29325|>=14.27.29016']", "vc[version='>=14.1,<15']", "vs2015_runtime[version='>=14.29.30037']", "vs2015_runtime[version='>=14.16.27012,<15.0a0|>=14.16.27033']", "vs2015_runtime[version='>=14.27.29016|>=14.28.29325|>=14.29.30037|>=14.32.31332']", "vc[version='14.*|9.*|>=14.1,<15.0a0|>=14,<15.0a0|>=9,<10.0a0']", "vc[version='>=14.2,<15']"}
Tests failed for tip-1.0.5-h587dcfd_0.tar.bz2 - moving package to C:\Users\isaac\miniconda3\envs\tipcforgebuild\conda-bld\broken
WARNING:conda_build.build:Tests failed for tip-1.0.5-h587dcfd_0.tar.bz2 - moving package to C:\Users\isaac\miniconda3\envs\tipcforgebuild\conda-bld\broken
TESTS FAILED: tip-1.0.5-h587dcfd_0.tar.bz2
```

Reply:
See[ my other comment ](https://github.com/conda-forge/staged-recipes/pull/20869#discussion_r1029680332)about manually grabbing the conda-forge pinnings from GitHub. Works for me.